scripts/airtable-to-json.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config()
+const fs = require('fs')
+
+const Airtable = require('airtable')
+
+Airtable.configure({
+  endpointUrl: 'https://api.airtable.com',
+  apiKey: process.env.AIRTABLE_API_KEY,
+})
+const base = Airtable.base(process.env.AIRTABLE_APP_ID)
+
+const recs = []
+base('GeorgeFloyd')
+  .select({
+    view: 'Grid view',
+    // maxRecords: 10, // Omit for unlimited
+  })
+  .eachPage(
+    function page(records, fetchNextPage) {
+      records.forEach((record) => {
+        const f = record.fields
+        recs.push({
+          id: f['TGD number'] || null,
+          city: f.City || null,
+          state: f.State || null,
+          tweet: f['Tweet URL'],
+          youtube: f.YouTube,
+          text: f['Doucette Text'],
```

Comment:
Nitpick: It might be nice to enforce the invariant that fields that have a value are strings, and those that do not have a value are `null`. So here we'd need to have `|| null` on the `youtube` and `text` too.

Furthermore, it's probably nice to trim the strings coming in from airtable in case a field is just whitespace. So I'd do this:

```js
function extractField(s) {
  return s?.trim() ?? null
}  
```

Reply:
Agreed. Landing this and filing a follow up